generate.sh

```diff
@@ -139,13 +140,32 @@ generateJavascript() {
   return 0
 }
 
+generatePython() {
+  LIBRARY="$1"
+  OPERATION="$2"
+  ARTIFACT_ID="symbol-openapi-$LIBRARY-client"
+  echo "Generating $LIBRARY"
+  rm -rf "$BUILD_DIR/$ARTIFACT_ID"
+  openapi-generator generate -g "$LIBRARY" \
+    -o "$BUILD_DIR/$ARTIFACT_ID" \
+    -t "$LIBRARY-templates/" \
+    -i "$INPUT" \
+    --additional-properties="projectName=$ARTIFACT_ID" \
+    --additional-properties="packageVersion=$VERSION" \
```

Comment:
`packageName` is `openapi_client`by default. 

https://openapi-generator.tech/docs/generators/python

Maybe we could change it to `symbol_openapi_client` so it look better when sdk or libs are importing the classes?

Reply:
Yes. Passed additional property packageName `symbol_openapi_client`.